documentation/assertions/any/to-satisfy.md

```diff
@@ -38,26 +44,46 @@ expected [ 0, 1, 2 ] to satisfy [ 0, 1 ]
 ]
 ```
 
-If you want to make assertions about the individual indexes in an array, you can
-do it the following way:
+In order to make statements about a subset of the available indices, an object
+specification on the right hand side can mention specific indexes as keys which
+are themselves compared using `to satisfy` semantics:
```

Comment:
Would be good with an example of a successful assertion here, too.

Reply:
I've added a success case example that shows full to satisfy semantics.